admin/assessmentExport.php

```diff
@@ -18,41 +18,60 @@
 {
   $cid = $_GET['cid'];
   
-  $assessment_array= array();  
-
-  $query = "SELECT imas_assessments.id,imas_assessments.name FROM `imas_assessments` 
-  join imas_courses 
-  ON imas_assessments.courseid = imas_courses.id
-  WHERE imas_courses.id = ".$cid .";" ;
+  $modules_array= array(); 
+  $assessment_names_array= array(); 
+  $module= loop1(1);
   
-  $result = mysql_query($query) or die("Query failed : " . mysql_error());
+  foreach ($module as $block) {
```

Comment:
If you end up building a page for selecting assessments to export, I'd recommend stealing code from /course/copyitems.php (lines 347-371 + 582-644), and you should be able to reuse the existing functions in /includes/copyiteminc.php.

If you have an item select page, then your processing/export script can just read the assessmentids from the POST.